zubhub_frontend/zubhub/src/components/notification_panel/NotificationPanel.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import styles from '../../assets/js/styles/components/notification_panel/notificationPanelStyles';
+import { makeStyles } from '@material-ui/core/styles';
+import NotificationPanelButton from './NotificationPanelButton';
+import cn from 'classnames';
+import NotificationPanelPopper from './NotificationPanelPopper';
+import { useMediaQuery } from '@material-ui/core';
+
+const useStyles = makeStyles(styles);
+
+const NOTIFICATION_VIEW_TYPE = {
+  ALL: 'ALL',
+  UNREAD: 'UNREAD',
+};
+
+const NotificationPanel = ({ open, anchorEl }) => {
+  const classes = useStyles();
+  const mediaQuery = useMediaQuery('(max-width: 600px)');
+  const [notificationViewType, setNotificationViewType] = useState(
+    NOTIFICATION_VIEW_TYPE.ALL,
+  );
+
+  const getAllNotificationView = () => {
+    const newNotificationsLength = 1;
+    const earlierNotificationsLength = 1;
+
+    return (
+      <div>
+        {newNotificationsLength > 0 && (
+          <h2 className={classes.panelSubheadingTextStyle}>New</h2>
+        )}
+        {earlierNotificationsLength > 0 && (
+          <h2 className={classes.panelSubheadingTextStyle}>Earlier</h2>
+        )}
+      </div>
+    );
+  };
+
+  const getUnreadNotificationView = () => (
+    <div>Unread notifications here...</div>
+  );
```

Comment:
Small thing but this doesn't show up under the Unread tab for the desktop view. It does show up under mobile though.